app-notifier/src/main/kotlin/org/ostelco/prime/appnotifier/FirebaseAppNotifier.kt

```diff
@@ -19,10 +20,15 @@ class FirebaseAppNotifier: AppNotifier {
 
     override fun notify(customerId: String, title: String, body: String) {
         println("Will try to notify customer with Id : $customerId")
-        sendNotification(customerId, title, body)
+        sendNotification(customerId, title, body, data = null)
     }
 
-    private fun sendNotification(customerId: String, title: String, body: String) {
+    override fun notify(customerId: String, title: String, body: String, data: Map<String, String>) {
+        println("Will try to notify-with-data customer with Id : $customerId")
```

Comment:
We should change this to use logger and not println. I have no idea why I did that in the first place :)

Reply:
done